.travis.yml

```
@@ -5,5 +5,18 @@ node_js:
   - "12"
   - "10"
 
+env:
+  - TESTS=true
+
+jobs:
+  fast_finish: true
+  include:
+    - node_js: 14
```

```suggestion
    - node_js: lts/*
```
no need to tie it to 14; this way it'll just always run on the latest LTS.